web/src/main/java/org/mskcc/cbio/oncokb/controller/DriveAnnotationParser.java

```diff
@@ -688,12 +672,14 @@ private void parseTherapeuticImplications(Gene gene, Set<Alteration> alterations
             addDateToLastEditSetFromObject(lastEditDates, drugObj, "name");
 //            addDateToLastReviewSetFromLong(lastReviewDates, drugObj, "name");
 
+            Pair<EvidenceType, String> evidenceTypeAndKnownEffect =  getEvidenceTypeAndKnownEffectFromDrugObj(drugObj);
+
             Evidence evidence = new Evidence();
-            evidence.setEvidenceType(evidenceType);
+            evidence.setEvidenceType(evidenceTypeAndKnownEffect.getKey());
             evidence.setAlterations(alterations);
             evidence.setGene(gene);
             evidence.setCancerTypes(new HashSet<>(tumorTypes));
-            evidence.setKnownEffect(knownEffectOfEvidence);
+            evidence.setKnownEffect(evidenceTypeAndKnownEffect.getValue());
```

Comment:
Maybe I can use `ImmutablePair` from apache library. The `getKey` and `getValue` method might be confusing. I just want to return two closely associated values.